### DISCUSSION TITLE
How to tag streets named after multiple people?

### DISCUSSION THREAD

**Discussion by mattsches** [EqualStreetNames/equalstreetnames#758] [Q&A] [answered]
Not an issue per se, and probably more an OSM-related topic: There are streets in our city that are named after _multiple_ people, e.g. after siblings. It's either explicit ("[Geschwister-Scholl-Straße](https://en.wikipedia.org/wiki/Hans_and_Sophie_Scholl)") or implicit (just a surname, but the municipal archiv suggests the street is named after two brothers).

According to the [OSM wiki](https://wiki.openstreetmap.org/wiki/Multiple_values), it could be tagged as `name:etymology:wikidata=value1;value2`, but there are alternative options, e.g. numbered suffix.

IMHO using the semicolon should be fine, I've tagged objects in this manner before in different contexts. But I suppose the processing scripts of this project wouldn't be able handle this correctly? I haven't taken a closer look, though.

What do you think?

I am happy to provide a PR if we decide on way to handle this.

**Comment by jbelien** [comment, marked as answer]
Multiple values are already handled in the data process ; separating the Wikidata items identifier with a semicolon in `name:etymology:wikidata` is indeed the way to go!

Here is an example in Brussels: https://www.openstreetmap.org/relation/3393466

**Comment by mattsches** [comment]
Wow, I really should have looked at the code first! Thanks for the quick answer and the example, we will map the affected streets accordingly. 🙂
